cha-ching-app/src/ElementsProviderCheckout.test.js

```diff
@@ -0,0 +1,45 @@
+import { render, screen } from "@testing-library/react";
+import ElementsProviderCheckout from "./ElementsProviderCheckout";
+import { MemoryRouter } from "react-router-dom";
+
+describe("<ElementsProviderCheckout/>", () => {
+  test("renders Loading when options are undefined", async () => {
+    const component = render(
+      <MemoryRouter initialEntries={[{ pathname: "/checkout", state: {} }]}>
+        <ElementsProviderCheckout />
+      </MemoryRouter>,
+    );
+    expect(await screen.findByRole("alert")).toBeInTheDocument();
```

Comment:
There's this role of "alert" here for the loading page. Could you link the Kent C. Dodds article if you find it again that showed you about this technique (for learning/off time reading)

Reply:
https://kentcdodds.com/blog/write-fewer-longer-tests